src/client/content_mapblock.cpp

```diff
@@ -585,7 +586,8 @@ void MapblockMeshGenerator::drawLiquidSides()
 			if (data->m_smooth_lighting)
 				color = blendLightColor(pos);
 			pos += origin;
-			vertices[j] = video::S3DVertex(pos.X, pos.Y, pos.Z, 0, 0, 0, color, vertex.u, v);
+			vertices[j] =
+					video::S3DVertex(pos.X, pos.Y, pos.Z, 0, 0, 0, color, vertex.u, v);
```

Comment:
Why line-break? Did this exceed the line length limit before?

Reply:
> Did this exceed the line length limit before?

Doesn't make sense, sorry. If it did, it still does, now.

Reply:
Oh, I just found the exact same thing https://github.com/minetest/minetest/pull/10029#discussion_r439428000. Didn't see this or I would have ignored it.

Reply:
Might be because GitHub uses very large tabs for indentation, whereas the configuration is for 4-space tabs, so appearance on GitHub is different.